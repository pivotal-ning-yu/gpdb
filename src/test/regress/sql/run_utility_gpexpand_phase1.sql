-- start_matchignore
-- m/.*-/
-- end_matchignore
\! echo 'EXPANSION_PREPARE_STARTED:<path> to inputfile' > $MASTER_DATA_DIRECTORY/gpexpand.status
\! gpcheckcat
\! gpconfig -r gp_debug_linger
\! gppkg
\! rm $MASTER_DATA_DIRECTORY/gpexpand.status
