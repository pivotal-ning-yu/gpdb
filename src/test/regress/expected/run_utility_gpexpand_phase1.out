-- start_matchignore
-- m/.*-/
-- end_matchignore
\! echo 'EXPANSION_PREPARE_STARTED:<path> to inputfile' > $MASTER_DATA_DIRECTORY/gpexpand.status
\! gpcheckcat
ERROR: Usage of gpcheckcat is not supported while the cluster is in a reconfiguration state, exit gpcheckcat
\! gpconfig -r gp_debug_linger
20190116:16:55:31:057863 gpconfig:zlv:gpadmin-[ERROR]:-ERROR: Usage of gpconfig is not supported while the cluster is in a reconfiguration state, exit gpconfig
\! gppkg
20190116:16:55:32:057884 gppkg:default-[ERROR]:-ERROR: Usage of gppkg is not supported while the cluster is in a reconfiguration state, exit gppkg
\! rm $MASTER_DATA_DIRECTORY/gpexpand.status
